main_test.go

```diff
@@ -360,3 +361,46 @@ func TestAlises(t *testing.T) {
 		}
 	}
 }
+
+func TestParseIniFile(t *testing.T) {
+	configHomeEnv := "XDG_CONFIG_HOME"
+	originalConfigHome := os.Getenv(configHomeEnv)
+	defer os.Setenv(configHomeEnv, originalConfigHome)
+
+	tests := []struct {
+		message       string
+		configPath    string
+		expectedError string
+	}{
+		{
+			message:       "valid ini file should parse correctly",
+			configPath:    path.Join("testdata", "ini", "valid"),
+			expectedError: "",
+		},
+		{
+			message:       "absent ini file should be ignored",
+			configPath:    path.Join("testdata", "ini", "missing"),
+			expectedError: "",
+		},
+		{
+			message:       "invalid ini file should raise error",
+			configPath:    path.Join("testdata", "ini", "invalid"),
+			expectedError: "couldn't read \"testdata/ini/invalid/yab/yab.ini\": \"testdata/ini/invalid/yab/yab.ini:2: time: unknown unit foo in duration 3foo\"",
```

Comment:
Instead of escaping quotes in strings, use the backticks string literal:

    `couldn't read "testdata/ini/invalid/yab/yab.ini": "..."`

Also, once you fix the `%q` for the error in the error message to use `%v`, this will be slightly different.